pkg/cloudencrypt/aws_test.go

```diff
@@ -0,0 +1,54 @@
+package cloudencrypt
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_AWSEncryptor(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	err := os.Setenv("AWS_ACCESS_KEY_ID", os.Getenv("TEST_AWS_ACCESS_KEY_ID"))
```

Comment:
Why do we prefix it with test? Why can't we use the `AWS_ACCESS_KEY_ID` right away?

Reply:
I just copied the approach from https://github.com/keboola/object-encryptor... We can adjust it.